resources/codegen/tests/conftest.py

```diff
@@ -0,0 +1,426 @@
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Shared fixtures for codegen tests
+"""
+
+# pylint: disable=invalid-name,redefined-outer-name
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+import logging
+import os
+import yaml
+
+import pytest
+
+from openassetio_codegen import datamodel
+
+
+@pytest.fixture(scope="package")
+def resources_dir():
+    """
+    The path to the resources directory for this test suite
+    """
+    test_dir = os.path.dirname(__file__)
+    return os.path.join(test_dir, "resources")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_all(resources_dir):
+    return os.path.join(resources_dir, "openassetio-codegen-test-all.yaml")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_traits_only(resources_dir):
+    return os.path.join(resources_dir, "openassetio-codegen-test-traits-only.yaml")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_specifications_only(resources_dir):
+    return os.path.join(resources_dir, "openassetio-codegen-test-specifications-only.yaml")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_invalid(resources_dir):
+    return os.path.join(resources_dir, "invalid.yaml")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_invalid_values(resources_dir):
+    return os.path.join(resources_dir, "invalid_values.yaml")
+
+
+@pytest.fixture(scope="package")
+def yaml_path_minimal(resources_dir):
+    return os.path.join(resources_dir, "minimal.yaml")
+
+
+@pytest.fixture(scope="package")
+def description_all(yaml_path_all):
+    with open(yaml_path_all, "r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
+
+@pytest.fixture(scope="package")
+def description_traits_only(yaml_path_traits_only):
+    with open(yaml_path_traits_only, "r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
+
+@pytest.fixture(scope="package")
+def description_specifications_only(yaml_path_specifications_only):
+    with open(yaml_path_specifications_only, "r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
+
+@pytest.fixture(scope="package")
+def description_invalid(yaml_path_invalid):
+    with open(yaml_path_invalid, "r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
+@pytest.fixture(scope="package")
+def description_invalid_values(yaml_path_invalid_values):
+    with open(yaml_path_invalid_values, "r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
```

Comment:
[fix] pep8
```suggestion


@pytest.fixture(scope="package")
def description_invalid_values(yaml_path_invalid_values):
    with open(yaml_path_invalid_values, "r", encoding="utf-8") as file:
        return yaml.safe_load(file)


```

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/529/commits/29227f9f1af08a0530d1d098cae95ae870b6fb59), with others.

Reply:
I believe the correct commit is 62f5d161 :+1: